### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by theopcoder** [theopcoder/PedestriaBotOfficial#22] [Ideas]
Post suggestions and ideas you would like to see added to the bot!

**Comment by MFHKiwi** [comment]
I'd like to see the auto-moderation feature become configurable as to what channels it affects. I've found myself trying to do things like `x person said y` and have it get removed by the bot.

**Reply by theopcoder** [reply]
Would this be for all auto moderation or just specific parts. Auto moderation consist (as of now) 4 features:

1. Log Deleted Messages
2. Discord Invite Protection
3. Chat Filter
4. Mute Bypass

Assuming you mean all auto moderation, all these would be disabled in the channels you "disable" them.

**Reply by MFHKiwi** [reply]
All, individually configurable.

**Reply by theopcoder** [reply]
As of now, I would only be able to have it exclude 1 channel per section. This means you can't exclude multiple channels

✅ 123456789
❌ 12345, 67890, 55555

Would this be ok?
